src/composable/ConditionalOrder.ts

```diff
@@ -189,23 +213,125 @@ export abstract class ConditionalOrder<Data, Params> {
    * A helper function for generically serializing a conditional order's static input.
    *
    * @param orderDataTypes ABI types for the order's data struct.
-   * @param staticInput The order's data struct.
+   * @param data The order's data struct.
    * @returns An ABI-encoded representation of the order's data struct.
    */
-  protected encodeStaticInputHelper(orderDataTypes: string[], staticInput: Data): string {
+  protected encodeStaticInputHelper(orderDataTypes: string[], staticInput: S): string {
     return utils.defaultAbiCoder.encode(orderDataTypes, [staticInput])
   }
 
   /**
-   * Apply any transformations to the parameters that are passed in to the constructor.
+   * Poll a conditional order to see if it is tradeable.
+   *
+   * @param owner The owner of the conditional order.
+   * @param p The proof and parameters.
```

Comment:
The proof isn't potentially passed in. Suggest that this be an optional parameter (array of strings).